telematic_system/scripts/log_analysis/log_timesheet.csv

```diff
@@ -0,0 +1,94 @@
+﻿Test Case,Run,Start Time,End Time

```

Comment:
This log timesheet doesn't need to be committed since it is specific to our testing. We can add a note in the read me about the expected format for this file